src/os_posix/os_fs.c

```diff
@@ -797,12 +797,16 @@ __posix_open_file(WT_FILE_SYSTEM *file_system, WT_SESSION *wt_session, const cha
 
     /* Create/Open the file. */
     WT_SYSCALL_RETRY(((pfh->fd = open(name, f, mode)) == -1 ? -1 : 0), ret);
-    if (ret != 0)
+    if (ret != 0) {
+        /* If we don't want error messages, just return the error value. */
+        if (F_ISSET(session, WT_SESSION_QUIET_TIERED) && ret == ENOENT)
+            goto err;
```

Comment:
It looks like setting `WT_SESSION_QUIET_TIERED` has the effect of suppressing any `ENOENT` errors.  So maybe name it accordingly, perhaps `WT_SESSION_QUIET_MISSING_FILES`?

Reply:
I had originally named it something like that, and then had a 2nd use that was similar but not ENOENT so I changed the name to be more generic. That 2nd use turned out not to be needed but I kept the generic naming. How strongly do you feel? I think there could be other places where we want to avoid errors for tiered and I'm not sure we want a separate quiet flag for each possible path/error value.

Reply:
I'm fine leaving it as QUIET_TIERED, seems like there's a good reason for it.